Makefile

```diff
@@ -69,3 +69,12 @@ kustomize_build:
 	# It will fail once we have more than one subdirectory, which will prevent us from
 	# adding untested configurations (this target will have to chage when that happens).
 	./kustomize build prometheus/* 1>/dev/null
+
+.PHONY: install_shellcheck
+install_shellcheck:
+	yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
```

Comment:
can you replace yum with dnf (just a suggestion)

Reply:
We've been using yum for this repo. I don't think we should have yum in one place and dnf in another. If we prefer to use dnf, then I think we should have everything changed to dnf across the repo at once.